Java/Rathna/Main.java

```diff
@@ -3,10 +3,79 @@
 public class Main {
```

Comment:
Maintaining different files will help you while revising these topics again in the future. Let's do that from next PR.